Code/Tools/AssetProcessor/native/resourcecompiler/rcjob.cpp

```diff
@@ -845,7 +845,9 @@ namespace AssetProcessor
                     }
                     else
                     {
-                        AZ_Warning(AssetBuilderSDK::WarningWindow, false, "Failed to find source path %s or temp path %s in non relative path in %s", sourceFile.c_str(), normalizedTempFolderPath.c_str(), product.m_productFileName.c_str());
+                        AZ_Warning(AssetBuilderSDK::WarningWindow, false,
```

Comment:
I know you didn't change this error message, but its hard to read. "Failed to find source path %s or temp path %s in non relative path in %s"

Its hard to read this and know what the problem is IMHO